samples/core/exit_handler/exit_handler_test.py

```diff
@@ -12,24 +12,52 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# %%
+
+import unittest
+from pprint import pprint
+
 import kfp
+import kfp_server_api
+
 from .exit_handler import pipeline_exit_handler
-from ...test.util import run_pipeline_func, TestCase
-
-run_pipeline_func([
-    TestCase(
-        pipeline_func=pipeline_exit_handler,
-        mode=kfp.dsl.PipelineExecutionMode.V1_LEGACY,
-    ),
-    # TODO(v2-compatible): The following test case fails with:
-    #   File "/Users/gongyuan/kfp/pipelines/sdk/python/kfp/compiler/compiler.py", line 678, in _create_dag_templates
-    #     launcher_image=self._launcher_image)
-    #   File "/Users/gongyuan/kfp/pipelines/sdk/python/kfp/compiler/v2_compat.py", line 108, in update_op
-    #     artifact_info = {"fileInputPath": op.input_artifact_paths[artifact_name]}
-    # KeyError: 'GCS path'
-    #
-    # TestCase(
-    #     pipeline_func=pipeline_exit_handler,
-    #     mode=kfp.dsl.PipelineExecutionMode.V2_COMPATIBLE,
-    # ),
-])
+from ...test.util import run_pipeline_func, TestCase, KfpMlmdClient
```

Comment:
nit: maybe import module instead of Class/func?

Reply:
good suggestion! I'll handle this later in separate PR